Dockerfile

```diff
@@ -11,7 +11,7 @@ USER root
 # https://github.com/bitnami/containers/pull/52661
 RUN groupadd -r spark && useradd -r -g spark spark_user
 
-RUN apt-get update && apt-get install -y \
+RUN apt-get update && apt-get install -y --no-install-recommends \
```

Comment:
Are you sure this has no negative effects? I could imagine a package having a recommended package that isn't required but speeds up certain operations, for example. Some python packages work that way

I'd be surprised if the recommended packages took a significant amount of space, is that the case?

Reply:
Sounds good. Reverted this change. It doesn't save us a lot of spaces either.

Reply:
:+1: